tools/tasks/seed/remove.debug.ts

```diff
@@ -0,0 +1,13 @@
+import * as gulp from 'gulp';
+import { join } from 'path';
+import Config from '../../config';
+const removeDebug = require('gulp-remove-logging');
+
+/**
+ * Remove all debug statements from distribution
+ */
+export = () => {
```

Comment:
Uglifyjs removes `debugger` statements. I'm not sure if we should remove `console.log` statements by default.